packages/plugin-aws-elasticsearch/__tests__/aws-elasticsearch.spec.ts

```diff
@@ -0,0 +1,35 @@
+const assert = require('assert');
+import { Microfleet } from '@microfleet/core';
+const { expect } = require('chai');
+
+describe('AWS Elasticsearch suite', () => {
+  let service: any;
+
+  it('should throw an error when plugin isn\'t included', async () => {
+    service = new Microfleet({
+      name: 'tester',
+      plugins: [],
+    });
+    assert(!service.awsElasticsearch);
+  });
+
+  it('able to connect to elasticsearch when plugin is included', async () => {
+    service = new Microfleet({
+      name: 'tester',
+      plugins: ['validator', 'logger', 'aws-elasticsearch'],
+    });
+
+    const [elastic] = await service.connect();
```

Comment:
its better to access property afterwards, check cassandra plugin for minimal case